### PR TITLE
feat: add secondary interface output address

### PR DESCRIPTION
**Pull request by jor2**
### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- When merging, use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

**Comment by jor2** [comment]
Auth policy issues blocking this PR.

File: outputs.tf [hunk 1]
```diff
@@ -20,12 +20,13 @@ output "list" {
...
-      id           = virtual_server.id
-      zone         = virtual_server.zone
-      ipv4_address = virtual_server.primary_network_interface[0].primary_ipv4_address
-      floating_ip  = var.enable_floating_ip ? ibm_is_floating_ip.vsi_fip[vsi_key].address : null
-      vpc_id       = var.vpc_id
+      name                   = virtual_server.name
+      id                     = virtual_server.id
+      zone                   = virtual_server.zone
+      ipv4_address           = virtual_server.primary_network_interface[0].primary_ipv4_address
+      secondary_ipv4_address = length(virtual_server.network_interfaces) == 0 ? "" : virtual_server.network_interfaces[0].primary_ipv4_address
```
Comment (ocofaigh) [inline outputs.tf:27]: Is there a reason you default to empty string instead of null when the length is 0?

**Review by ocofaigh — APPROVED** [review]

**Comment by ocofaigh** [comment]
Pipeline failed with `Error: [ERROR] Error creating authorization policy: The policy wasn't created because an access policy with identical attributes already exists. Please update the roles in the existing policy (75e0e543-f538-450e-a27b-3ce460919c35), or update the one you're trying to assign to include a different attribute assignment.`
@jor2 you probably need to manually delete this auth policy from our account so this can pass

**Comment by jor2** [comment]
> Pipeline failed with `Error: [ERROR] Error creating authorization policy: The policy wasn't created because an access policy with identical attributes already exists. Please update the roles in the existing policy (75e0e543-f538-450e-a27b-3ce460919c35), or update the one you're trying to assign to include a different attribute assignment.` @jor2 you probably need to manually delete this auth policy from our account so this can pass

passing

**Pull request merged**